data/mods/gen2/scripts.js

```diff
@@ -601,13 +602,19 @@ let BattleScripts = {
 			if (move.crit) {
 				level *= 2;
 			}
+			this.add('hint', 'Gen 2 Present has a glitched damage calculation using the secondary types of the Pokemon for the Attacker\'s Level and Defender\'s Defense.', true);
```

Comment:
Here and below I'm kind of torn because the `-hint` would read better if it came after the `damage` line, but I'm not sure if there's an easy way to do it and complicating things to change the ordering of the `-hint` subtly seems like the wrong decision? But I could be missing something?